pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -293,15 +293,15 @@ func (alloc *Action) predicate(task *api.TaskInfo, node *api.NodeInfo) ([]*api.S
 	if ok, resources := task.InitResreq.LessEqualWithResourcesName(node.FutureIdle(), api.Zero); !ok {
 		return nil, api.NewFitError(task, node, api.WrapInsufficientResourceReason(resources))
```

Comment:
Here and `prePredict` still return api.NewFitError, we should unify them.

Reply:
NewFitError will set the status code with Error as default.

Reply:
I mean both `predict` and `prePredict` should return same value.